src/client/app/audit/audit.component.ts

```diff
@@ -7,19 +7,20 @@ import { Observable } from 'rxjs/Observable';
 import * as _ from 'lodash';
 import * as moment from 'moment';
 
+import { UserService } from 'app/admin/user';
+import {
+	PagingOptions, TableSortOptions,
+	SortDisplayOption, SortDirection
+} from 'app/shared';
 
 import { AuditService } from './audit.service';
 import { AuditViewDetailModalContext, AuditViewChangeModal, AuditViewDetailModal } from './audit-view-change.component';
-import { PagingOptions } from '../shared/pager.component';
-import { TableSortOptions } from '../shared/pageable-table/pageable-table.component';
-import { SortDisplayOption, SortDirection } from '../shared/result-utils.class';
-import { UserService } from '../admin/users.service';
 import { AuditOption } from './audit.classes';
 
 
 @Component({
 	selector: 'audit',
-	templateUrl: './audit-list.component.html'
+	templateUrl: './audit-list.component.html',
```

Comment:
It was adopted in ES5.1, and it doesn't work in function parameters. There is a [proposal](https://github.com/tc39/proposal-trailing-function-commas) for it though.